.github/workflows/build-artifacts-code.yml

```
@@ -63,4 +63,4 @@ jobs:
             src/out/*
       - name: Publish Nuget packages to GitHub registry
         working-directory: src
-        run: dotnet nuget push out/* -k ${{secrets.GITHUB_TOKEN}}
+        run: dotnet nuget push "out/*.nupkg" -k ${{secrets.GITHUB_TOKEN}}
```

```suggestion
        run: dotnet nuget push "out/*" -k ${{secrets.GITHUB_TOKEN}}
```

This may sound funny, but we can now even remove the file extension.

---

Do as you see fit. I like to see extension explicitly.